segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/WriteOperation.java

```diff
@@ -164,6 +171,19 @@ private void logEnd() {
         val elapsed = timer.getElapsed();
         SLTS_WRITE_LATENCY.reportSuccessEvent(elapsed);
         SLTS_WRITE_BYTES.add(length);
+        SLTS_NUM_CHUNKS_ADDED.reportSuccessValue(chunksAddedCount.get());
+        if (segmentMetadata.isStorageSystemSegment()) {
+            SLTS_SYSTEM_WRITE_LATENCY.reportSuccessEvent(elapsed);
+            SLTS_SYSTEM_WRITE_BYTES.add(length);
+            SLTS_SYSTEM_NUM_CHUNKS_ADDED.reportSuccessValue(chunksAddedCount.get());
+            val name = segmentMetadata.getName().substring(segmentMetadata.getName().lastIndexOf("/") + 1).replace("$", "_");
```

Comment:
here too. 

Also, is there an opportunity to factor this block of code out? Looks very similar to the one above.

Reply:
done